samples/app-dev/Connectors/Miscellaneous/ODataS4hana/readme.md

```diff
@@ -0,0 +1,253 @@
+# OData with SAP S4 Hana Example
+
+
+## Description
+
+This example demonstrates how we can create SAP S4 Hana connection using OData connector and use all OData CRUD activities for SAP S4 Hana server.
+OData app with SAP S4 Hana bascially contain 2 activities i.e. OData Query and OData Modify. OData Modify contains three operation i.e. Create, Update and Delete. 
+The main purpose of these activities are:
+a) To fetch SAP S4 Hana data using OData Query activity.
+b) To create data in SAP S4 Hana server using Create operation of OData Modify activity.
+c) To update data in SAP S4 Hana server using Update operation of OData Modify activity.
+d) To delete data in SAP S4 Hana server using Delete operation of OData Modify activity.
+
+## Prerequisites
+
+1. Ensure that SAP S4 Hana services are up and running
+  
+## Import the sample
+
+1. Download the sample json file i.e. ODataS4HanaCRUD.json.
+
+2. Create a new empty app
+
+![Create an app](../../../import-screenshots/ODataS4Hana/60.png)
+
+3. On the app details page, select import app option.
+
+![Select import](../../../import-screenshots/ODataS4Hana/61.png)
+
+4. Now click on ‘browse to upload’ button and select the app.json from your machine that you want to import.
+
+![Import your sample](../../../import-screenshots/ODataS4Hana/1.png)
+
+5. Click on Upload Button. The Import app dialog displays some generic errors and warnings as well as any specific errors or warnings pertaining to the app you are importing.
+
+![The Import app dialog](../../../import-screenshots/ODataS4Hana/2.png)
+
+6. In Import App dialog, there are 2 options for connection:
+
+* Create new connection – If you are choosing this option then it will create new connection.
+
+* Existing connections – If you are choosing this option then it will use existing connections from the environment.
+
+7. After importing app is done, in connection tab make sure to re-enter the password and click on connect button to establish the connection.
+
+## Understanding the configuration
+
+### The Connection
+
+When you import the app you need to re-enter the password and establish the connection.
+
+![The connection](../../../import-screenshots/ODataS4Hana/3.png)
+
+In the connection, note that,
+1. Root URL - In this field we give root URL of OData services (i.e. SAP S4 Hana, Dataverse).
+2. Username - Username to connect to SAP S4 Hana.
+3. Password - Password to connect to SAP S4 Hana.
+4. Authorization Type - You can choose the authorization type from the following options:
+
+		 	None: No Username and Password is required for aunthentication.
+		 	Basic: Basic authentication using Username and Password.
+		 	OAuth2: Access the token based client authentication by using OAuth 2.0 athentication service.
+		 
+Authorization type None 
+
+![The connection](../../../import-screenshots/ODataS4Hana/4.png)
+
+Authorization type Basic
+
+![The connection](../../../import-screenshots/ODataS4Hana/5.png)
+
+Authorization type OAuth2
```

Comment:
You can remove this along with screenshot as this Authorization type is covered in Dataverse.